.github/CODEOWNERS

```diff
@@ -0,0 +1,40 @@
+# Centrifudge Code Owners
+#
+# Order is important; the last matching pattern takes the most
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+*       @branan @mikiquantum @mustermeiszer @offerijns
+
+# Changes to the service of our chain.
+src/*    @mikiquantum
+
+# Changes to pallets
+pallets/*  @branan @mikiquantum @mustermeiszer
+
+## Changes to specific pallets
+### Connectors related logic
+pallets/connectors/* @NunoAlexandre
+
+### Rewards related logic
+pallets/rewards/* @lemunozm
+pallets/liquidity-rewards/* @lemunozm
+
+### Nft-sales related logic
+pallets/nft-sales/* @NunoAlexandre
+
+### Pallet Keystore
+pallets/keystore/* @cdamian
+
+# Changes to libs
+libs/* @branan @mikiquantum @mustermeiszer
```

Comment:
Here could have sections where any other than these three can review it (in my case, fees, ensure, rewards modules, i.e). Is it needed to express that level too?

Reply:
Yes, that would be needed. Tell me what you want and I add it

Reply:
`fees`, `ensure`, `rewards` modules for me in `cfg_traits` 👍🏻